cla_backend/apps/call_centre/tests/api/test_case_api.py

```diff
@@ -285,6 +286,106 @@ def test_patch_provider_notes_not_allowed(self):
         self.assertNotEqual(response.data["provider_notes"], "abc123")
 
 
+class SuggestProviderTestCase(BaseCaseTestCase):
+    #     # new tests to check assign_suggest view
+    #     # this post always returns a response with the format:
+    #     # {"suggested_provider": *, "as_of": "date_time",
+    #     #  "suitable_providers":
+    #     #  [{"name": "*", "id": pk, "short_code": "*",
+    #     "telephone_frontdoor": "*", "telephone_backdoor": "0845 122 8677"},
+    #     #  ...]}
+
+    def setUp(self):
+        super(CLAOperatorAuthBaseApiTestMixin, self).setUp()
+        category1 = make_recipe("legalaid.category")
+        category2 = make_recipe("legalaid.category")
+
+        cat1_provider1 = make_recipe("cla_provider.provider", active=True)
+        make_recipe(
+            "cla_provider.provider_allocation", weighted_distribution=0.5, provider=cat1_provider1, category=category1
+        )
+
+        cat1_provider2 = make_recipe("cla_provider.provider", active=True)
+        make_recipe(
+            "cla_provider.provider_allocation", weighted_distribution=0.5, provider=cat1_provider2, category=category1
+        )
+
+        cat2_provider1 = make_recipe("cla_provider.provider", active=True)
+        make_recipe(
+            "cla_provider.provider_allocation", weighted_distribution=0.5, provider=cat2_provider1, category=category2
+        )
+        cat2_provider2 = make_recipe("cla_provider.provider", active=True)
+        make_recipe(
+            "cla_provider.provider_allocation", weighted_distribution=0.5, provider=cat2_provider2, category=category2
+        )
+
+        helper1 = ProviderAllocationHelper()
+        self.suggested_cat_providers = helper1.get_qualifying_providers(category1)
```

Comment:
My IDE flags issues with some of the Instance attributes in suggested_cat_providers defined.

These instances need fields within the class

```
def __init__(self):
   self.suggested_cat_providers = None
   self.suggested_category = None
   self.cat_eligibility_check = None
```